docroot/sites/all/modules/features/forms/forms.install

```diff
@@ -98,6 +98,19 @@ function forms_update_7007() {
 }
 
 /**
+ * 8/29/2016: Delete form submissions from two forms:
```

Comment:
Caroline, the Drupal convention is to include short, descriptive comments on update hooks. Drush outputs the message before running the update hook, as you've probably seen. Please modify this message and amend the commit to follow the convention, then force push to github. Thanks!